.storybook/main.js

```diff
@@ -21,4 +26,7 @@ module.exports = {
     '@storybook/preset-create-react-app',
   ],
   staticDirs: ['../public'],
+  features: {
+    storyStoreV7: true,
```

Comment:
For [On-demand story loading](https://storybook.js.org/docs/react/configure/overview#on-demand-story-loading)